components/FlexibleHeader/src/MDCFlexibleHeaderView.h

```diff
@@ -329,6 +329,14 @@ IB_DESIGNABLE
  */
 @property(nonatomic) BOOL minMaxHeightIncludesSafeArea;
 
+/**
+ A layout guide that equates to the top safe area inset of the flexible header view.
+
+ Use this layout guide to position subviews in the flexible header in relation to the top safe area
+ insets.
+ */
+@property(nonatomic, nonnull, readonly) id topSafeAreaGuide;
```

Comment:
id?  Why can't have be a type / protocol?
If this is because iOS 8 doesn't support NSLayoutGuides, please document that here.

Reply:
Added docs indicating how this object is intended to be used. NSLayoutConstraint's API accepts an `id`, thus this type being an id along with the fact that we may switch to a UILayoutGuide once we drop iOS 8 support.